### PR TITLE
[maintenance]Define charset in database url

### PR DESCRIPTION
**Pull request by Ferror**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | master           |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets | none                     |
| License         | MIT                                                          |

Change made regarding https://github.com/Sylius/Sylius/pull/13786

### CODE REVIEW

File: .github/workflows/application.yml [hunk 1]
```diff
@@ -315,7 +315,7 @@ jobs:
 
         env:
             APP_ENV: test_cached
-            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?serverVersion=mariadb-${{ matrix.mariadb }}"
+            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=mariadb-${{ matrix.mariadb }}
```
Comment (coldic3) [inline .github/workflows/application.yml:318]:
```suggestion
            DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?charset=utf8mb4&serverVersion=mariadb-${{ matrix.mariadb }}"
```

**Review by coldic3 — APPROVED** [review]

**Comment by lchrusciel** [comment]
requires rebase

**Comment by Ferror** [comment]
@lchrusciel done :)

**Review by lchrusciel — COMMENTED** [review]
There some sort of syntax issue with pipeline 
image

Can you take a look?

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Zbigniew! :1st_place_medal:

**Pull request merged**